move-wsl

```diff
@@ -41,7 +41,7 @@ fi
 
 # Export WSL image to tar file
 echo "Exporting VHDX to \"$TEMP_FILE\" ..."
-wsl --export $WSL_NAME $TEMP_FILE
+wsl --export $WSL_NAME $TEMP_FILE.part && mv "$TEMP_FILE.part" "$TEMP_FILE"
```

Comment:
will it be better to handle the return value of "wsl --export" instead of creating a temp file for the temp file? The PowerShell version does that for the export since it's a critical step.

Reply:
@sidecus Sounds like a good idea.

Reply:
> will it be better to handle the return value of "wsl --export" instead of creating a temp file for the temp file? The PowerShell version does that for the export since it's a critical step.

Agreed. I didn't really know how to handle this specific error so I leveraged the existing "missing file" handling. I'll copy the logic from the PowerShell version instead, thanks for mentioning that.